topo/node/srl/srl.go

```diff
@@ -37,12 +41,65 @@ func (n *Node) ConfigPush(ctx context.Context, ns string, r io.Reader) error {
 	return status.Errorf(codes.Unimplemented, "Unimplemented")
 }
 
-func (n *Node) CreateNodeResource(_ context.Context, _ node.Interface) error {
-	return status.Errorf(codes.Unimplemented, "Unimplemented")
+func (n *Node) CreateNodeResource(ctx context.Context, ni node.Interface) error {
+	log.Infof("Creating Srlinux node resource %s", n.pb.Name)
+
+	srl := srl.Srlinux{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Srlinux",
+			APIVersion: "kne.srlinux.dev/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: n.pb.Name,
+			Labels: map[string]string{
+				"app":  n.pb.Name,
+				"topo": ni.Namespace(),
+			},
+		},
+		Spec: srl.SrlinuxSpec{
+			NumInterfaces: len(ni.Interfaces()),
+			Config: &srl.NodeConfig{
+				Sleep: 0,
```

Comment:
what does sleep do?

Reply:
that is the delay in seconds before starting the pod, as per `kne` config

https://github.com/google/kne/blob/570083e95d00fcd4f9ab5e5ceb3a1c95b63c6ee1/proto/topo.proto#L79